index.php

```diff
@@ -10,10 +10,58 @@
 </head>
 
 <body>
-    <div class="col-12">
-        <center>
-            <h2>Roam Radience Travelers</h2>
-        </center>
+    <div class="container-fluid">
+        <div class="row">
+            <?php include 'navbar.php'; ?>
+            <!-- main banner -->
+            <div class="col-12 mt-3">
+                <div class="card text-bg-dark banner-image-contanier">
+                    <img src="resources/images/hero-image.jpg" class="card-img banner-image" alt="...">
+                    <div class="card-img-overlay">
+                        <h3 class="card-title">Free up your mind</h3>
+                        <p class="card-text">Let's travel</p>
+                        <button class="btn banner-image-button">Explore Journeys</button>
+                    </div>
+                </div>
+            </div>
+            <!-- main banner -->
+            <!-- category cards -->
+            <div class="col-12">
+                <div class="row">
+                    <div class="col-12 col-md-4 col-lg-4 mt-3">
+                        <div class="card category-card">
+                            <img src="resources/images/login.jpg" class="card-img-top category-card-img" alt="...">
```

Comment:
Don't put empty **alt** tag like this. You have to put some suitable alt text to all image tags. Its more important in SEO process (google crawlers).